WordPress/Classes/WordPress.xcdatamodeld/WordPress 31.xcdatamodel/contents

```diff
@@ -248,6 +248,7 @@
         <attribute name="title" optional="YES" attributeType="String">
             <userInfo/>
         </attribute>
+        <attribute name="videopressGUID" optional="YES" attributeType="String" syncable="YES"/>
```

Comment:
Yeah, I figured we'd have to resolve that after 5.3 is released. We could cherry pick the commits or bring develop current. @sendhil if you want me to work on merging the branch let me know. :smile: